test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -181,10 +216,31 @@ if [ "$FLOW_AGGREGATOR" == "true" ]; then
             ssh -F ssh-config k8s-node-control-plane kubectl apply -f elk-flow-collector/elk-flow-collector.yml -n elk-flow-collector
             LOGSTASH_CLUSTER_IP=$(ssh -F ssh-config k8s-node-control-plane kubectl get -n elk-flow-collector svc logstash -o jsonpath='{.spec.clusterIP}')
             ELK_ADDR="${LOGSTASH_CLUSTER_IP}:4739:udp"
-
             $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $ELK_ADDR > "${FLOW_AGG_YML}"
-        else
-            $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -fc $FLOW_COLLECTOR > "${FLOW_AGG_YML}"
+
+        elif [[ $FLOW_COLLECTOR == "GRAFANA" ]]; then
+            echo "Deploy Grafana flow collector"
+            pushImgToNodes "$GRAFANA_IMG_NAME" "$SAVED_GRAFANA_IMG"
+            pushImgToNodes "$CH_SERVER_IMG_NAME" "$SAVED_CH_SERVER_IMG"
+            pushImgToNodes "$CH_MONITOR_IMG_NAME" "$SAVED_CH_MONITOR_IMG"
+            pushImgToNodes "$CH_OPERATOR_IMG_NAME" "$SAVED_CH_OPERATOR_IMG"
+            pushImgToNodes "$METRICS_EXPORTER_IMG_NAME" "$SAVED_METRICS_EXPORTER_IMG"
+
+            # generate manifest and push to nodes
+            $THIS_DIR/../../../../hack/generate-manifest-flow-visibility.sh --mode dev > "${FLOW_VIS_YML}"
+            $THIS_DIR/../../../../hack/generate-manifest-flow-aggregator.sh --mode dev -ch > "${FLOW_AGG_YML}"
+            cp "${CH_OPERATOR_INSTALL_BUNDLE_BASE_YML}" "${CH_OPERATOR_INSTALL_BUNDLE_YML}"
+            copyManifestToNodes "$FLOW_VIS_YML"
+            copyManifestToNodes "$CH_OPERATOR_INSTALL_BUNDLE_YML"
```

Comment:
if `CH_OPERATOR_INSTALL_BUNDLE_BASE_YML` is not modified before copying to nodes, copying to /tmp (`CH_OPERATOR_INSTALL_BUNDLE_YML`) seems redundant to me. see ANTREA_IPSEC_YML

Reply:
You're right. I've deleted it